packages/catalogue/metadatasearch/bundle.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @class Oskari.catalogue.bundle.metadatasearch.MetadataSearchBundle
+ *
+ */
+Oskari.clazz.define("Oskari.catalogue.bundle.metadatasearch.MetadataSearchBundle", function() {
+
+}, {
+	"create" : function() {
+
+		return Oskari.clazz.create("Oskari.catalogue.bundle.metadatasearch.MetadataSearchBundleInstance");
+
+	},
+	"start" : function() {
+	},
+	"stop" : function() {
+	},
+	"update" : function(manager, bundle, bi, info) {
+
+	}
+}, {
+
+	"protocol" : ["Oskari.bundle.Bundle", "Oskari.bundle.BundleInstance", "Oskari.mapframework.bundle.extension.ExtensionBundle"],
+	"source" : {
+
+		"scripts" : [
+			/* service */
+			{
+	            "type": "text/javascript",
+	            "src": "../../../bundles/catalogue/metadatasearch/service/MetadataOptionService.js"
+	        }, {
+	            "type": "text/javascript",
+	            "src": "../../../bundles/catalogue/metadatasearch/service/MetadataSearchService.js"
+	        },
```

Comment:
These could be just imported on instance.js

Reply:
Or rather they already are imported in the handler but both are ES classes so referencing them here probably does nothing but add dead code :)